ui/packages/catalog-ui-search/src/main/webapp/js/model/Workspace.js

```diff
@@ -90,8 +90,13 @@ module.exports = Backbone.AssociatedModel.extend({
         this.listenTo(this, 'change', this.handleChange);
         this.listenTo(this, 'error', this.handleError);
     },
-    handleListChange: function() {
-        this.set('saved', false);
+    handleListChange: function(model) {
+        if (model !== undefined &&
+            _.intersection(Object.keys(model.changedAttributes()), [
+                'actions'
+            ]).length === 0) {
```

Comment:
The condition after the `&&` seems like it could be simplified to `!model.changedAttributes().hasOwnProperty('actions')`.

Reply:
I'm not sure. I just copied and modified the code from `handleChange()`. @andrewkfiedler would the suggested code work?

Reply:
What if `actions` _and_ another attribute changed? In that case we'd still want the workspace to indicate that it has unsaved changes. Is that situation possible?